quickstart/pom.xml

```diff
@@ -56,6 +56,7 @@
     <dependency>
       <groupId>io.rest-assured</groupId>
       <artifactId>rest-assured</artifactId>
+      <version>4.2.0</version>
```

Comment:
Remind me: what was the reason why we had to hard-code the version here? We could maybe add a comment explaining this.

Reply:
It is not provided in the quarkus bom. I can extract it to a parameter if you want

Reply:
Hmm I remember there was something else... @absurdfarce do you remember? But for now let's keep it the way it is, it's just a detail.